src/ImportDefinitionsBundle/Export/Exporter.php

```diff
@@ -213,6 +244,8 @@ private function exportRow(DefinitionInterface $definition, Concrete $object, $p
         {
             $data[$mapItem->getFromColumn()] = $this->getObjectValue($object, $mapItem, $data, $definition, $params, $runner);
         }
+        
+        $provider->addExportData($data, $configuration, $definition, $params);
```

Comment:
I add the data to the provider here instead of returning it. I feel the data should be passed on to the provider before the object is considered exported, before the export_definition.object.finished event is dispatched.